From 9ffcbbe994de4eb13d997739398807b3af3a746a Mon Sep 17 00:00:00 2001
From: Liming Wang <liming.wang@windriver.com>
Date: Thu, 5 Jul 2012 15:31:21 +0800
Subject: [PATCH 23/28] SPEAr: kill off __mem_pci

commit 5621caac ARM: kill off __mem_pci

Removes all __mem_pci defines of arm, so also delete it for spear.

Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 arch/arm/plat-spear13xx/include/plat/io.h |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/plat-spear13xx/include/plat/io.h b/arch/arm/plat-spear13xx/include/plat/io.h
index 4d4ba82..6fd43cb 100644
--- a/arch/arm/plat-spear13xx/include/plat/io.h
+++ b/arch/arm/plat-spear13xx/include/plat/io.h
@@ -17,6 +17,5 @@
 #define IO_SPACE_LIMIT		0xFFFFFFFF
 
 #define __io(a)			__typesafe_io(a)
-#define __mem_pci(a)		(a)
 
 #endif /* __PLAT_IO_H */
-- 
1.7.9.7

